.github/workflows/js-css-coding-standards.yml

```diff
@@ -0,0 +1,43 @@
+name: JS and CSS Coding Standards
+
+on:
+  push:
+    branches: [ trunk, develop ]
+  pull_request:
+    branches: [ trunk, develop ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Adding Node.js setup to run JS and CSS linters
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18.15'  # project version
+
+      - name: Get npm cache directory
+        id: npm-cache
+        run: echo "::set-output name=dir::$(npm config get cache)"
+
+      - name: Cache npm dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            **/node_modules
```

Comment:
Where did you get the idea to cache the `node_modules` specifically? All tutorials I can find on this just use the `outputs.dir` folder from the line below 🤔

Reply:
Well, that was my idea, not sure if this is needed, I can remove it.

Reply:
I'd remove it based on this (taken from [these docs](https://github.com/actions/cache/blob/main/examples.md#node---npm)):

> Note It is not recommended to cache node_modules, as it can break across Node versions and won't work with npm ci

Reply:
I'm wondering if it wouldn't be better to simply use the `cache` directive in `actions/setup-node` directly: https://github.com/marketplace/actions/setup-node-js-environment#caching-global-packages-data 

It's supposed to use `actions/cache` under the hood and is guaranteed to work and to simplify our entire action 🤔